### PR TITLE
DNN-10714 - Clear date interval from Datepicker component

### PR DESCRIPTION
**Pull request by ricardodiasc**

### CODE REVIEW

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/SearchPage/SearchAdvancedDetails.jsx [hunk 1]
```diff
@@ -108,6 +108,7 @@ class SearchAdvancedDetails extends Component {
                                 onDayClick={this.props.onDayClick}
                                 dropdownIsActive={this.state.DropdownCalendarIsActive}
                                 applyChanges={() => {this.toggleDropdownCalendar(); this.props.onApplyChangesDropdownDayPicker();}}
+                                clearChanges={() => {this.toggleDropdownCalendar();this.props.clearAdvancedSearchDateInterval();}}
```
Comment (tpluscode) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/SearchPage/SearchAdvancedDetails.jsx]: Please don't do such one-liners. Let's keep each expression in its own line

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/DropdownDayPicker/DropdownDayPicker.jsx [hunk 2]
```diff
@@ -66,7 +66,10 @@ class DropdownDayPicker extends Component  {
                                             onDayClick={(data) => onDayClick(data, true) }/>
                                 </GridCell>
                                 <GridCell columnSize={100}>
-                                    <Button type="primary" onClick={()=>applyChanges()}>Apply</Button>
+                                    <div style={{"width": "100%", "text-align": "center", "height": "30%"}}>
+                                        <Button type="secondary" style={{"margin-right":"5px"}} onClick={()=>clearChanges()}>Clear</Button>
+                                        <Button type="primary" onClick={()=>applyChanges()}>Apply</Button>
```
Comment (tpluscode) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/DropdownDayPicker/DropdownDayPicker.jsx]: Aren't those **Clear**/**Apply** labels coming from a resource file?

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/DropdownDayPicker/DropdownDayPicker.jsx [hunk 3]
```diff
@@ -66,7 +66,10 @@ class DropdownDayPicker extends Component  {
                                             onDayClick={(data) => onDayClick(data, true) }/>
                                 </GridCell>
                                 <GridCell columnSize={100}>
-                                    <Button type="primary" onClick={()=>applyChanges()}>Apply</Button>
+                                    <div style={{"width": "100%", "text-align": "center", "height": "30%"}}>
+                                        <Button type="secondary" style={{"margin-right":"5px"}} onClick={()=>clearChanges()}>Clear</Button>
```
Comment (tpluscode) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/DropdownDayPicker/DropdownDayPicker.jsx]: Would it be possible to move inline styles to the stylesheet?

**Review by tpluscode — CHANGES_REQUESTED** [review]

**Comment by kanm2015** [comment]
code review done

**Pull request merged**